client/src/modules/dashboard/Events/pages/EditEventPage.tsx

```diff
@@ -58,6 +58,7 @@ export const EditEventPage: NextPage = () => {
         capacity: parseInt(String(data.capacity)),
         start_at: data.start_at,
         ends_at: data.ends_at,
+        url: data.url?.trim() ?? null,
```

Comment:
```suggestion
        url: data.url?.trim() || null,
```
If the `url` is just `''` then `??` will return it and then the validation will fail.